### ISSUE TITLE
method "interactive" in PIXI.Sprite "undefined"

### ISSUE THREAD

**Issue by rtalon83** [finscn/pixi.js#3]
There are several missing methods in Sprite.

-interactive
-buttonMode

**Comment by finscn** [comment]
because When I build pixi , I exclude some module.
you have to build dist js by your self, don't use the js file I built

**Comment by rtalon83** [comment]
Yes. I did a new library with the pixi-source 4.7 and your "lights" xD. Thanks
